applications/pelion_client/README.rst

```diff
@@ -0,0 +1,568 @@
+.. _pelion_client:
+
+nRF Pelion Client
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Pelion Client application demonstrates the integration of the Pelion Device Management library (Pelion DM library) within the |NCS|.
```

Comment:
We okay with shortenign the name like this when it's against the naming convention of the product on which it's based?

Reply:
Fixed.